libs/e2e/src/lib/describer/modelAssessment/Constants.ts

```diff
@@ -131,5 +131,30 @@ export enum Locators {
   ModelOverviewMetricChart = "#ModelOverview #ModelOverviewMetricChart",
   ModelOverviewMetricChartBars = "#ModelOverview #ModelOverviewMetricChart .highcharts-bar-series > rect",
   MissingParameterPlaceholder = "#ModelOverview #MissingParameterPlaceholder",
-  CounterfactualHeader = "#ModelAssessmentDashboard span:contains('What-If counterfactuals')"
+  CounterfactualHeader = "#ModelAssessmentDashboard span:contains('What-If counterfactuals')",
```

Comment:
great to see these tests!

Reply:
Thanks for calling them out! 😊
First time writing UI tests and it was an interesting experience...